db/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy import Column, Integer, String
+
+
+class Base(DeclarativeBase): pass
```

Comment:
немного наследования
Добавь в класс Base поле id
```suggestion
class Base(DeclarativeBase): 
    id = Column(Integer, primary_key=True)
```
а в модели `User` убери этот id, таким образом атрибут id будет доступен в классе `User`, тк ты унаследовал `Base` и все его атрибуты. id всегда и во всех таблицах будет - это стандартный первичный ключ таблицы, Пожэтому остальные таблицы наследуем от Base и в моделях не указываем id